.github/workflows/ci_pr.yml

```diff
@@ -43,24 +51,24 @@ jobs:
           - python-version: 2.7
             PYLINTOPTS: "--rcfile=ci/2.7.pylintrc --ignore=tests_e2e,makepkg.py"
 
-          - python-version: 3.4
```

Comment:
should we keep python 3.4? we should check telemetry

Reply:
They are very low and around 2000 vms running on py 3.4. All of them are either ubuntu 14 or Debian 8